.github/scripts/cmd/cmd.py

```diff
@@ -284,6 +511,54 @@ def main():
             print_and_log('❌ Failed to generate prdoc')
             sys.exit(exit_code)
 
+    elif args.command == 'label':
+        # The actual labeling is handled by the GitHub Action workflow
+        # This script validates and auto-corrects labels
+
+        try:
+            # Check if PR is still open and not merged/in merge queue
+            pr_number = os.environ.get('PR_NUM')
+            if pr_number:
+                if not check_pr_status(pr_number):
+                    raise ValueError("Cannot modify labels on merged PRs or PRs in merge queue")
+
+            # Check if user has permission to modify labels
+            is_org_member = os.environ.get('IS_ORG_MEMBER', 'false').lower() == 'true'
+            is_pr_author = os.environ.get('IS_PR_AUTHOR', 'false').lower() == 'true'
+
+            if not is_org_member and not is_pr_author:
```

Comment:
Shouldn't this if condition be OR instead of and as it mentioned in the error message below?

Reply:
When you are not an org member AND not the pr author, you are authorized. Expressed differently, when you are an org member it resolves to false. If you are not an org member, it checks if you are the author.